### DISCUSSION TITLE
[Question]: minimum configuration if deployed on a cloud server?

### DISCUSSION THREAD

**Discussion by SuperBen118** [danny-avila/LibreChat#1889] [Q&A]
### What is your question?

If deployed on a cloud server, what are the minimum configuration requirements needed? Please provide a detailed description, as the official README does not mention this.

### More Details

Urgent, waiting online.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fuegovic** [comment]
I believe 1vCPU / 2GB should be enough

**Comment by xueshanf** [comment]
Were are the uploaded files stored? They can be large, right? Are they in MongoDB or VectorDB?
